src/index.ts

```diff
@@ -1,5 +1,3 @@
-import * as types from './types'
-import * as pcd from './pcd'
-import * as utils from './utils'
-
-export { types, pcd, utils }
```

Comment:
Why we remove this line?

Reply:
> Because it caused a type error once imported as `pcd`. And this way it's easier to import any exported elements from the package.
> 
> ```
> Type 'typeof import("/Users/yanismeziane/Desktop/anon-aadhaar/build/src/pcd")' does not satisfy the constraint 'PCDPackage<any, any, any, any>'.
>   Property 'name' is missing in type 'typeof import("/Users/yanismeziane/Desktop/anon-aadhaar/build/src/pcd")' but required in type 'PCDPackage<any, any, any, any>'.ts(2344)
> pcd.ts(34, 3): 'name' is declared here.
> ```

Ah! Make sense! Thank you for your explanation.